src/ipa_pc/data_structures.rs

```diff
@@ -339,28 +343,19 @@ impl<G: AffineCurve> PCPreparedCommitment<Commitment<G>> for PreparedCommitment<
 pub struct Randomness<G: AffineCurve> {
     /// Randomness is some scalar field element.
     pub rand: Vec<G::ScalarField>,
-
-    /// Randomness applied to the shifted commitment is some scalar field element.
-    pub shifted_rand: Option<G::ScalarField>,
 }
 
 impl<G: AffineCurve> PCRandomness for Randomness<G> {
     fn empty(segments_count: usize) -> Self {
         Self {
-            rand: vec![G::ScalarField::zero(); segments_count],
-            shifted_rand: None,
+            rand: vec![G::ScalarField::zero(); segments_count]
         }
     }
```

Comment:
that trait is not necessary if we have an affine curve associated to PCCommitment (or the committer key)
Let's get rid of it and let us use the UniformRand implementation for the ScalarField 
(Maybe that needs CanonicalSerialize and Deserialized to be added to the Field trait.)

Reply:
Rand function is removed from trait, but "empty" function is used in generic implementations in lib.rs so for now I cannot remove it